source/common/quic/quic_stat_names.cc

```diff
@@ -28,15 +28,18 @@ void QuicStatNames::chargeQuicConnectionCloseStats(Stats::Scope& scope,
                                                    bool is_upstream) {
   ASSERT(&symbol_table_ == &scope.symbolTable());
 
+  if (error_code >= quic::QUIC_LAST_ERROR) {
+    ENVOY_LOG(warn, fmt::format("Error code {} is out of range of QuicErrorCodes.", error_code));
```

Comment:
Can you add a test case for this? Also, I think warning here can log spam. I would recommend making this debug level and potentially also adding a stat for unknown error code or something like that. Thank you.

/wait

Reply:
done.
I utilized quic::QUIC_LAST_ERROR_CODE as a stat for all out of range codes.